recipes/ckan-deploy.rb

```diff
@@ -109,7 +113,7 @@
 	mode "0755"
 	variables({
 		:app_name =>  app['shortname'],
-		:app_url => app['domains'][0],
+		:app_url => node['datashades']['ckan_web']['site_domain'],
```

Comment:
could we not pull the config json from ssm param store?

Reply:
We could. But what is the advantage of Parameter Store in this use case?

Application URLs and version numbers don't need the encryption that Parameter Store provides, and they don't need to be writeable by the server. They don't need to be available at runtime. We don't need to update them in the same way that we can toggle Mailtrap on and off. The amount of data is fixed and relatively small, so we don't need Parameter Store's scalability.

Chef Custom JSON has the advantage of being injected into our deployment context so it's already available as a variable in our recipes, no lookup needed, and no extra IAM permissions to permit the lookup. It's also easier to generate, because we can populate the CloudFormation attribute using YAML (which is much more Jinja-template-friendly than JSON) and it's automatically converted.